chsdi/templates/htmlpopup/ivs_nat.mako

```diff
@@ -30,7 +30,11 @@
     <tr><td class="cell-left">${_(ivs_slaname)}</td><td>${c['attributes']['ivs_slaname']}</td></tr>
     <tr><td class="cell-left">${_('ivs_documentation')}</td>
 <%
-    from urllib2 import urlopen
```

Comment:
it makes no sense just to import libraries without using them. I would just remove the import